OpenEdXMobile/src/main/java/org/edx/mobile/discussion/CommentBody.java

```diff
@@ -0,0 +1,41 @@
+package org.edx.mobile.discussion;
+
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+public class CommentBody {
+    private String threadId;
+    private String rawBody;
+    private String parentId;
+
+    public CommentBody(@NonNull String threadId, @NonNull String rawBody, @Nullable String parentId) {
+        this.threadId = threadId;
+        this.rawBody = rawBody;
+        this.parentId = parentId;
+    }
+
+    public void setParentId(@Nullable String parentId) {
```

Comment:
We should define getters and setters whenever they will be needed, we can remove them in this class as they are un-used currently.